ftp/src/test/java/docs/javadsl/FtpTraversingExample.java

```diff
@@ -6,16 +6,18 @@
 
 // #traversing
 import akka.NotUsed;
+import akka.stream.Materializer;
 import akka.stream.alpakka.ftp.FtpFile;
 import akka.stream.alpakka.ftp.FtpSettings;
 import akka.stream.alpakka.ftp.javadsl.Ftp;
 import akka.stream.javadsl.Source;
 
 public class FtpTraversingExample {
 
-  public Source<FtpFile, NotUsed> listFiles(String basePath, FtpSettings settings)
+  public void listFiles(String basePath, FtpSettings settings, Materializer materializer)
       throws Exception {
-    return Ftp.ls(basePath, settings);
+    Ftp.ls(basePath, settings)
+        .runForeach(ftpFile -> System.out.println(ftpFile.toString()), materializer);
```

Comment:
Not quite sure about this one: on the one hand it is nice to show in the example that this produces a source of `FtpFile`, on the other hand I think it is good to show actually running the stream, not just creating it, especially in 'early' examples.

Reply:
I agree. The FTP examples are from the early days of Alpakka...